pages/about.html

```diff
@@ -1,12 +1,36 @@
-<!DOCTYPE html>
+<html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Mac Peyton's Personal Portfolio Site</title>
+    </head>
+    <body class="about-me">
+        <header>
+            <h1>Mac Peyton (they | them)</h1>
+            <h2> Aspiring Back-End Developer.</h2>
+            <h3> QT Dreamer, Activist, Possibility Creator.</h3>
+        </header>
+        <nav>
+            <a href="index.html">Home</a>
+            <a href="portfolio.html">Code Examples</a>
+        </nav>
+        <p>
+            A little bit about me: 
+            <ul>
+                <li>I am in school to become a full-stack developer, though my passion is for back-end development.</li>
+                <li>I have a Masters in Social Work which allows me to think critically in my problem solving and come up with diverse solutions in coding.</li>
+                <li>I grew up in Utah and now live in Washington.</li>
+                <li>I love being in nature (hiking, rock climbing, camping), spending time with community, and participating in activism.</li>
+                <li>I make shit happen.</li>
+                <li>I am going to edit all of this once it actually goes live. Right now I'm just practicing for class.</li> 
+            </ul>
+        </p>
```

Comment:
Consider using a tag other than `p` here, it's perhaps more of a `section` than a paragraph.